.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,39 @@
+# All SDK Contribution checklist:
+
+This checklist is used to make sure that common guidelines for a pull request are followed.
+- [ ] Please add REST spec PR link to the SDK PR
+- [ ] **Please open PR in `Draft` mode if it is:**
+	- Work in progress or not intended to be merged.
+	- Encountering multiple pipeline failures and working on fixes.
+- [ ] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/master/CONTRIBUTING.md).**
+- [ ] **The pull request does not introduce [breaking changes](https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/breaking-change-rules.md).**
+
+### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/master/CONTRIBUTING.md#general-guidelines)
+- [ ] Title of the pull request is clear and informative.
+- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).
+
+### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/master/CONTRIBUTING.md#testing-guidelines)
+- [ ] Pull request includes test coverage for the included changes.
+
+### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/master/CONTRIBUTING.md#sdk-generation-guidelines)
+- [ ] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
+- [ ] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
```

Comment:
Is this really a requirement? I don’t think we want them manually adding a commit id

Reply:
Those are track 2/data plane requirement